s30161-pj-NikodemCyrzan/zajecia1/instalacja-debiana-linux.md

```diff
@@ -0,0 +1,523 @@
+
+-   Ext2 file system - szybki system plików bez księgowania. Księgowanie to odnotowywanie aktualnie trwającej operacji, dzięki czemu w razie przerwania pracy systemu na przykład z powodu braku prądku, system plików nie zostaie uszkodzony. Z powodu braku tej funkcji system narażony jest na awarie.
+-   Ext3 jurnaling file system - system plików podobny do Ext2, ale z księgowaniem. Z powodu księgowania jest wolniejszy niż Ext2. Jest wstecznie kompatybilny z Ext2.
+-   Ext4 jurnaling file system - system plików który jest szybszy od Ext2 pomimo posiadania księgowania. Nie jest wstecznie kompatybilny.
+-   btrfs jurnaling file system - system plików posiadający księgowanie i działający jak system kontroli wersji.
+-   JFS jurnaling file system - 64-bitowy system plików posiadający księgowanie. Był to jeden z pierwszych systemów z księgowaniem, ale współcześnie nie ma potrzeby używania go, ponieważ nowsze systemy są wydajniejsze od niego.
+-   XFS jurnaling file system - 64-bitowy system plików posiadający księgowanie. Jest bardzo wydajny w przypadku zapisu i odczytu wielkich plików. Umożliwia również równoległą pracę na plikach, dzięki czemu jednocześnie może być prowadzony zapis i odczyt.
+-   FAT16 file system - system plików posługujący się 16-bitowym adresowaniem, które ograniczało pojemność dysków do 32 MB.
+-   FAT32 file system - system plików posługujący się 32-bitowym adresowaniem wykorzystywany często w dyskach przenośnych jak Pendrive, czy dyskach zewnętrznych.
+-   swap area - dodatkowa przestrzeń na dysku służąca jako zapasowa pamięć RAM. Przestrzeń swap powinna być 2 razy większa niż pamięć RAM, jeśli jest jej mniej niż 2GB. W przeciwnym wypadku powinno być jej tyle co pamięć RAM + 2 GB.
+-   physical volume for encryption - partycja z przeznaczeniem do szyfrowania danych.
+-   physical volume for RAID - partycja z przeznaczeniem do tworzenia macierzy RAID, umożliwiających zwiększanie wydajności i niezawodoności dysków przechowujących dane.
+-   physical volume for LVM - partycja z przeznaczeniem do użycia z systemem LVM, do zarządzania przestrzenią dyskową.
+
+### Konfiguracja menadżera paczek
+
+Ważne jest wybranie jak najbliższego archiwum, aby skrócić niepotrzebne opóźnienia.
+
+> Mój wybór
+>
+> `Poland, ftp.task.gda.pl`, ponieważ jest to archiwum znajdujące się na Politechnice Gdańskiej, czyli dosyć blisko.
+
+### Kolekcje oprogramowania które można wybrać:
+
+-   Debian desktop environment - zawiera ogólne pliki umożliwiające obsługę GUI.
+-   GNOME - najpopularniejsze GUI używane na systemach linux.
+-   Xfce - bardzo lekkie i szybkie GUI.
+-   GNOME Flashback - szybsza i lżejsza wersja GNOME.
+-   KDE Plasma - bardzo przyjazne dla użytkownika GUI, z wieloma animacjami i ładnym designem.
+-   Cinnamon, MATE, LXDE, LXQt - GUI bardzo podobne do systemu Windows.
+-   web server - serwer HTTP.
+-   SSH server - serwer SSH.
+-   Standard system utilities - podstawowe narzędzia takie jak `doc-debian`, `ftp`, `python` i więcej.
```

Comment:
> i więcej

Czyli co konkretnie?
Czy system bez tego będzie działał i czy będzie można go sensownie używać?
A jeśli nie to czego będzie brakowało i czy można to doinstalować później?

Lista lub link.
Do uzupełnienia.